README.md

```diff
@@ -1 +1,11 @@
-# Hello-world
\ No newline at end of file
+# Hello world
+This project helps me practice the Github Flow which includes
+1. Create a branch
+2. Create a pull request
+3. Address review comments
+4. Marge your request
+5. Delete your branch
+
+Languages used are
+1. Html
+2. Css
```

Comment:
- [ ] it would be better if you use the [Readme Template](https://github.com/microverseinc/readme-template) and customize it to your own you should have a section where you tell how to use this project locally and other relevant section for this project

Reply:
I just want to be clear
do you mean i should use the exact formate in the Readme Template even if I don't have all those details.

Reply:
format I mean

Reply:
@Linushaddai99 you shouldn't delete all the section keep them add you information like social media , how to install the project , etc